lib/util/exec/common.ts

```diff
@@ -14,6 +14,7 @@ export enum BinarySource {
 
 export interface ExecConfig {
   binarySource: Opt<BinarySource>;
+  dockerChildPrefix: Opt<string>;
```

Comment:
```suggestion
  dockerChildPrefix: string;
```
Shouldn't be null or undefined.